CreeDictionary/CreeDictionary/templates/CreeDictionary/components/metatags.html

```diff
@@ -11,7 +11,7 @@
 
   {% load url_extras %}
 
-  {% with title_="itwêwina: the online Cree dictionary" desc_="A smart, online dictionary of Plains Cree (Y-Dialect) | nêhiyawi-masinahikan kâ-iyiniwâk" %}
+  {% with title_="itwêwina: the online Cree dictionary" desc_="A smart, online dictionary of Plains Cree (Y-Dialect) | nêhiyawêwasinahikan kâ-nipwâhkâmakahk" %}
```

Comment:
Would this not be `nêhiyawasinahikan`?

Reply:
That's what I thought too, but this is the translation we got from our source. However, I'm not merging this until you can lookup both words in the dictionary!

Reply:
Interesting! I wonder if the different would be "Cree book" and "Cree speaking book" 🤔